pandas/tests/indexes/test_datetimelike.py

```diff
@@ -0,0 +1,174 @@
+"""
+Tests shared for DatetimeIndex/TimedeltaIndex/PeriodIndex
+"""
+from datetime import datetime, timedelta
+
```

Comment:
should these be in pandas/tests/indexes/datetimelike/test_equals (or as appropriate)?

Reply:
havent decided.  im split between implementing tests/indexes/{index_type}/methods/ vs tests/indexes/methods/ for things like equals, leaning towards the latter.

Reply:
yeah doing this for other types, so i think it makes sense here

Reply:
do you mind holding off on that move while i get a little closer to getting everything collected

Reply:
kk, ok for merge here, but really think this is inconcsistent